tests/e2e/utils/playwright-setup.js

```diff
@@ -343,6 +343,7 @@ export const setupWoo = async () => {
 		'wp config set WP_DEBUG false --raw',
 		'wp plugin install woocommerce --force --activate',
 		'wp plugin install woocommerce-gateway-stripe --activate',
+		'wp plugin install disable-emails --activate',
```

Comment:
This was a good idea 💯 I'd just add a comment saying what's the purpose of this installation to avoid someone removing it in the future.